src/request.rs

```diff
@@ -140,3 +143,32 @@ impl DispatchSignedRequest for Client {
 
     }
 }
+
+#[derive(Debug, PartialEq)]
+pub struct TlsError {
+    message: String
+}
+
+impl Error for TlsError {
+    fn description(&self) -> &str {
+        &self.message
+    }
+}
+
+impl fmt::Display for TlsError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.message)
+    }
+}
+
+/// Helper method for creating an http client with tls
+pub fn make_http_client_with_tls() -> Result<Client, TlsError> {
```

Comment:
I violently dislike this method name :)  

Especially if it's going to be part of the default incantation to get all services, I think we should name it something shorter without `make` and `with` like `default_tls_client` or `default_request_dispatcher` and document what it provides with method comments.